cmd/apic/template.go

```diff
@@ -42,11 +43,12 @@ func init() {
 }
 
 type templateVerb struct {
-	Dir        string        `help:"The output directory"`
-	Tracer     string        `help:"The template function trace expression"`
-	Gopath     string        `help:"the go path to use when looking up packages"`
-	GlobalList []string      `help:"A global value setting for the template"`
-	Search     file.PathList `help:"The set of paths to search for includes"`
+	Dir            string        `help:"The output directory"`
+	Tracer         string        `help:"The template function trace expression"`
+	Gopath         string        `help:"the go path to use when looking up packages"`
+	GlobalList     []string      `help:"A global value setting for the template"`
+	Search         file.PathList `help:"The set of paths to search for includes"`
+	TemplateSearch file.PathList `help:"Search path apic includes"`
```

Comment:
I found this help string a bit hard to parse, maybe mirror the previous one:
`The set of paths to search for templates`